lib/libvfio-user.c

```diff
@@ -468,29 +468,39 @@ consume_fd(int *fds, size_t nr_fds, size_t index)
 
 int
 handle_dma_map(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg,
-               struct vfio_user_dma_region *region)
+               struct vfio_user_dma_map *region)
```

Comment:
can we rename "region" now too? to "dma_map" or something?

Reply:
sure